src/Our.Umbraco.Ditto/ComponentModel/Processors/HtmlStringProcessor.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Web;
```

Comment:
It's doable.  Were you thinking that it would replace one of the Umbraco core's PVC for the RTE?

The core ones can be easily removed with this code...

    PropertyValueConvertersResolver.Current.RemoveType<RteMacroRenderingValueConverter>();